tests/Unit/Evolution/Systems/ScalarWave/Test_Equations.cpp

```diff
@@ -51,21 +51,31 @@ void check_du_dt(const size_t npts, const double time) {
   }();
 
   auto box = db::create<db::AddSimpleTags<
-      Tags::dt<ScalarWave::Pi>, Tags::dt<ScalarWave::Phi<Dim>>,
-      Tags::dt<ScalarWave::Psi>, ScalarWave::Pi,
+      ScalarWave::Tags::ConstraintGamma2, Tags::dt<ScalarWave::Pi>,
+      Tags::dt<ScalarWave::Phi<Dim>>, Tags::dt<ScalarWave::Psi>, ScalarWave::Pi,
+      ScalarWave::Phi<Dim>,
       Tags::deriv<ScalarWave::Pi, tmpl::size_t<Dim>, Frame::Inertial>,
+      Tags::deriv<ScalarWave::Psi, tmpl::size_t<Dim>, Frame::Inertial>,
       Tags::deriv<ScalarWave::Phi<Dim>, tmpl::size_t<Dim>, Frame::Inertial>>>(
       Scalar<DataVector>(pow<Dim>(npts), 0.0),
+      Scalar<DataVector>(pow<Dim>(npts), 0.0),
       tnsr::i<DataVector, Dim, Frame::Inertial>(pow<Dim>(npts), 0.0),
       Scalar<DataVector>(pow<Dim>(npts), 0.0),
       Scalar<DataVector>(-1.0 * solution.dpsi_dt(x, time).get()),
+      [&x, &time, &solution ]() noexcept {
+        return solution.dpsi_dx(x, time);
+      }(),
       [&x, &time, &solution ]() noexcept {
         auto dpi_dx = solution.d2psi_dtdx(x, time);
         for (size_t i = 0; i < Dim; ++i) {
           dpi_dx.get(i) *= -1.0;
         }
         return dpi_dx;
       }(),
+      [&x, &time, &solution ]() noexcept {
+        auto dpsi_dx = solution.dpsi_dx(x, time);
+        return dpsi_dx;
+      }(),
```

Comment:
Same question (and if the lambda is needed... why not just `return solution...` like above?)

Reply:
its not really needed, removed it